[2023-09-04T06:02:32.874+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-02T00:00:00+00:00 [queued]>
[2023-09-04T06:02:32.889+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-02T00:00:00+00:00 [queued]>
[2023-09-04T06:02:32.890+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:02:32.922+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-04-02 00:00:00+00:00
[2023-09-04T06:02:32.932+0000] {standard_task_runner.py:57} INFO - Started process 5375 to run task
[2023-09-04T06:02:32.936+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-04-02T00:00:00+00:00', '--job-id', '456', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmprtc1euyg']
[2023-09-04T06:02:32.939+0000] {standard_task_runner.py:85} INFO - Job 456: Subtask insert_into_table
[2023-09-04T06:02:33.022+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-02T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:02:33.184+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-02T00:00:00+00:00'
[2023-09-04T06:02:33.187+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-04-02', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:02:33.210+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:33.226+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:33.243+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-04-02', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:02:33.249+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:02:33.270+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220402T000000, start_date=20230904T060232, end_date=20230904T060233
[2023-09-04T06:02:33.310+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:02:33.463+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:05:16.893+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-02T00:00:00+00:00 [queued]>
[2023-09-04T06:05:16.909+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-02T00:00:00+00:00 [queued]>
[2023-09-04T06:05:16.909+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:05:16.931+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-04-02 00:00:00+00:00
[2023-09-04T06:05:16.940+0000] {standard_task_runner.py:57} INFO - Started process 6664 to run task
[2023-09-04T06:05:16.945+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-04-02T00:00:00+00:00', '--job-id', '1001', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpjqkl7nuq']
[2023-09-04T06:05:16.949+0000] {standard_task_runner.py:85} INFO - Job 1001: Subtask insert_into_table
[2023-09-04T06:05:17.034+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-04-02T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:05:17.187+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-02T00:00:00+00:00'
[2023-09-04T06:05:17.189+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-04-02', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:05:17.203+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:17.216+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:05:17.238+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-04-02', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:05:17.245+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:05:17.269+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220402T000000, start_date=20230904T060516, end_date=20230904T060517
[2023-09-04T06:05:17.319+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:05:17.439+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
